File: july18_hw/1220A/code.py
from collections import Counter
n =int(input())
s = list(str(input()))

    
freq = Counter(s)
cnt0 = freq["z"]
cnt1 = freq["n"]
for i in range(cnt1):
    print(1,end=" ")
for i in range(cnt0):
    print(0,end=" ")

File: july18_hw/1220A/code.java
import java.util.Scanner;
// cards problem
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n=sc.nextInt();
        String s=sc.next();
        char[] chars = s.toCharArray();
        
        // make freq list from chars 
        int cnt0 =0;
        int cnt1 =0;
        for (char ch : chars){
            if (ch== 'z'){
                cnt0++;
            }
            if (ch== 'n'){
                cnt1++;
            }
        }
        for (int i=0;i<cnt1;i++){
            System.out.println(1+" ");
        }
        for (int i=0;i<cnt0;i++){
            System.out.println(0+" ");
        }
    }
}
